src/index.js

```diff
@@ -8,6 +8,7 @@ import RedBox from './components/RedBox';
 import View from './components/View';
 import Text from './components/Text';
 import TextStyles from './sharedStyles/TextStyles';
+import Symbol from './symbol';
```

Comment:
@jongold fwiw, i think the difference here is that "Artboard" isn't a JavaScript-specific concept, but "Symbol" is - and I think that language-specific construct names are always automatically off limits. You shouldn't have a "Map" either - a "GoogleMap" or a "StreetMap" or something, but not a "Map" - that's taken by the language.